crates/biome_cli/src/execute/traverse.rs

```diff
@@ -466,9 +473,23 @@ fn process_messages(options: ProcessMessagesOptions) {
                         let severity = diag.severity();
                         if severity == Severity::Error {
                             *errors += 1;
+                            let should_print = printed_error_diagnostics < max_diagnostics;
+                            if should_print {
```

Comment:
Shadowing could be reduced.
```suggestion
                            if printed_error_diagnostics < max_diagnostics {
```

Reply:
Thank you!
https://github.com/biomejs/biome/pull/1248/commits/a2d9c0f7ee28d993961b8f5daefb7eaa017b200a ← fixed it!